bokeh/palettes.py

```diff
@@ -1224,7 +1224,7 @@ def gray(self, n):
         return self.linear_palette(self.Greys256, n)
 
 # need to explicitly transfer the docstring for Sphinx docs to build correctly
-_mod = _PalettesModule('bokeh.palettes')
+_mod = _PalettesModule(str('bokeh.palettes'))
```

Comment:
Why this `str()` is needed?

Reply:
it's not yet but the boilerplate top code that all modules will has includes `unicode_literals` from `__future__`